modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/SettingsFragmentPage.kt

```diff
@@ -229,6 +232,15 @@ private fun AutoAddToUpNextRow(onClick: () -> Unit) {
     )
 }
 
+@Composable
+private fun HeadphoneControlsRow(onClick: () -> Unit) {
+    SettingRow(
+        primaryText = stringResource(LR.string.settings_title_headphone_controls),
+        icon = GradientIconData(IR.drawable.ic_headphone),
```

Comment:
Should this be a tinted icon as we aren't using a gradient?

Reply:
`GradientIcon` handles empty colors list internally so ignoring this one.